lib/events/Event.ts

```diff
@@ -133,14 +137,16 @@ export namespace EventMocks {
     },
     shouldHideAfterStartDate: false,
     userAttendeeStatus: "attending",
-    userMilesFromEvent: 12.7892
+    userMilesFromEvent: 12.7892,
+    attendeeCount: 3
   } as CurrentUserEvent
 
   export const Multiday = {
```

Comment:
Can you give these mock events to @nairoldi and @tristan8chu, they can use them to test the backend